frontend/src/components/maps/leaflet/InfoSlideOut/InfoSlideOut.tsx

```diff
@@ -165,36 +235,42 @@ const InfoControl: React.FC<InfoControlProps> = ({ open, setOpen, onHeaderAction
           <InfoButton
             id="slideOutInfoButton"
             variant="outline-secondary"
-            onClick={setOpen}
+            onClick={() => {
+              if (!open) {
+                setOpen();
```

Comment:
Not a big deal, but I don't like how setOpen acts as a toggle, but we are explicitly using it in cases where we **know** that we want it closed, or open, so I'd rather we pass the desired state in setOpen.

Reply:
Ooooh I see what you mean

Reply:
I could rename it for clarity?

Reply:
wait I think I got it, nvm